src/Teams/Models/TeamMembers.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Teams.Models
+{
+    [Table("team_members")]
+    public partial class TeamMembers
+    {
+        [Key]
+        public int Id { get; set; }
+        [Required]
+        [Column("team_id")]
+        [StringLength(450)]
+        public string TeamId { get; set; }
```

Comment:
Same as described for Team entity. Please do not use partial classes as they will add additional complexity to the code.